### ISSUE TITLE
Passphrase

### ISSUE THREAD

**Issue by mightydanp** [LukeShortCloud/winesapOS#572]
When booting from usb is says that 
"A password is required to access the cryptroot volume" 
Ive tried the passwords you have listed none of them work

**Comment by mightydanp** [comment]
Looking throw the document can you please add the word password as a log in in the documentation where you have specified what passwards are. It took me 15 minutes to figure out the paswords was password

**Comment by LukeShortCloud** [comment]
We have this documented here:

https://github.com/LukeShortCloud/winesapOS/tree/3.2.1#secure-image

I'll try to make this more prominent. Perhaps putting this information on the GitHub releases page will also help.

**Comment by LukeShortCloud** [comment]
@mightydanp Thanks for all of your suggestions! I've updated our README.md documentation with the following changes:

- Header for explaining the different images was changed from "Differences" to now be "[Differences Between Performance, Secure, and Minimal Images](https://github.com/LukeShortCloud/winesapOS/tree/test#differences-between-performance-secure-and-minimal-images)".
- A new section called "[Passwords](https://github.com/LukeShortCloud/winesapOS#passwords)".
    - Added a note here the `root` password needs to  be changed immediately.

Let me know if you have any other suggestions and I'd be happy to look into it!
